unseal.go

```diff
@@ -140,11 +140,16 @@ func (k *SealedKeyObject) UnsealFromTPM(tpm *TPMConnection, pin string) ([]byte,
 		return nil, err
 	}
 
+	// Required for metadata version > 0
```

Comment:
it this is a irrelevant for version 0 or needs to be guarded?  The comment should be more of the form

// For metadata version > 0 we set the pin as auth value , that's how we in that case auth is in ....

Reply:
I've updated this. It doesn't need to be guarded for version 0 because without the TPM2_PolicyAuthValue assertion, the pin doesn't get included in the session HMAC and so this is a noop (see https://github.com/canonical/go-tpm2/blob/master/cmds_ea.go#L366, https://github.com/canonical/go-tpm2/blob/master/auth.go#L300 and https://github.com/canonical/go-tpm2/blob/master/auth.go#L142